engine/src/main/java/org/terasology/logic/characters/CharacterSoundSystem.java

```diff
@@ -133,6 +144,33 @@ public void onLanded(VerticalCollisionEvent event, EntityRef entity, CharacterSo
         }
     }
 
+    /**
+     * React to any horziontal collison of a character that has a
+     * component for character sounds and decide what the volume
+     * of the landing sound should be and the sounds should pe played.
+     * Then send a PlaysSoundEvent
+     * @param event the event that triggered the horizontal collision
+     * @param entity the entity that collide horizontal
+     * @param characterSounds the component that contains info about character sounds
+     */
+    @ReceiveEvent
+    public void onHorrizontalCollision(HorizontalCollisionEvent event, EntityRef entity, CharacterSoundComponent characterSounds) {
+        Vector3f horizVelocity = new Vector3f(event.getVelocity());
+        horizVelocity.y = 0;
+        float velocity = horizVelocity.length();
+
+        if (velocity > HORIZONTAL_DAMAGE_SPEED_THRESHOLD) {
```

Comment:
I noticed that this was initially a reference to the `healthComponent.horizontalDamageSpeedThreshold` field, but has now been made a class-level constant -- can you elaborate on the reasons for this change?